packages/zoe/test/unitTests/setupMixedMints.js

```diff
@@ -0,0 +1,39 @@
+import harden from '@agoric/harden';
+
+import produceIssuer from '@agoric/ertp';
+
+const setupMixed = () => {
+  const ccBundle = produceIssuer('CryptoCats', 'strSet');
+  const moolaBundle = produceIssuer('moola');
+  const allBundles = { cc: ccBundle, moola: moolaBundle };
+  const mints = new Map();
+  const issuers = new Map();
+  const amountMaths = new Map();
+  const brands = new Map();
+
+  for (const k of Object.getOwnPropertyNames(allBundles)) {
```

Comment:
I really like how you did this! It takes the best of the array version and the best of the bundle version.